core/lib/spree/app_configuration.rb

```diff
@@ -390,13 +404,20 @@ def payment_canceller
     # returns a String
     class_name_attribute :taxon_url_parametizer_class, default: 'ActiveSupport::Inflector'
 
+    ##
+    # Logger for solidus
+    # @return [Logger] The logger which by default outputs to STDOUT.
+    #
+    def logger
+      @logger ||= Logger.new(STDOUT)
+    end
+
```

Comment:
I'm not sure if it's worth keeping a separate logger for Solidus only, and that be different than the `Rails.logger`

Reply:
I think this should be a decision for the developers. Some developers prefer to keep their logs aggregated. However I think it would be ok to default it to `Rails.logger` instead of creating a new logger by default